frontend/src/store/credential.js

```diff
@@ -0,0 +1,218 @@
+//
+// SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import {
+  defineStore,
+  acceptHMRUpdate,
+} from 'pinia'
+import {
+  computed,
+  reactive,
+} from 'vue'
+
+import { useApi } from '@/composables/useApi'
+
+import { isOwnSecret } from '@/utils'
+
+import { useAuthzStore } from './authz'
+import { useAppStore } from './app'
+import { useGardenerExtensionStore } from './gardenerExtension'
+import { useCloudProfileStore } from './cloudProfile'
+
+import filter from 'lodash/filter'
+import isEmpty from 'lodash/isEmpty'
+import set from 'lodash/set'
+import get from 'lodash/get'
+import some from 'lodash/some'
+
+function namespaceNameKey ({ namespace, name }) {
+  return `${namespace}/${name}`
+}
+
+export const useCredentialStore = defineStore('credential', () => {
+  const api = useApi()
+  const appStore = useAppStore()
+  const authzStore = useAuthzStore()
+  const gardenerExtensionStore = useGardenerExtensionStore()
+  const cloudProfileStore = useCloudProfileStore()
+
+  const state = reactive({
+    secretBindings: {},
+    secrets: {},
+    quotas: {},
+  })
+
+  const isInitial = computed(() => {
+    return isEmpty(state.secretBindings)
+  })
```

Comment:
I think this is only used in a test. can we drop it?

Reply:
removed